cmake/modules/autopas.cmake

```diff
@@ -34,12 +35,17 @@ if(ENABLE_AUTOPAS)
     add_library(libautopas IMPORTED STATIC GLOBAL)
     add_dependencies(libautopas autopas)
 
+    # Using target_compile_definitions for imported targets is only possible starting with cmake 3.11, so we use add_definitions here.
+    if(OPENMP)
+        add_definitions(-DAUTOPAS_OPENMP)
```

Comment:
Just for clarification: why do you need this line? Doesn't line 28 already add the definition to autopas? Or do you want this macro also to be defined in mardyn?